src/main/webapp/resources/js/pages/analysis/components/AnalysisJsonPreview.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * @File component renders a JSON preview of output files.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Divider, Row, Typography } from "antd";
+import { getDataViaChunks } from "../../../apis/analysis/analysis";
+import { ContentLoading } from "../../../components/loader/ContentLoading";
+import { BasicList } from "../../../components/lists/BasicList";
+import { SPACE_XS } from "../../../styles/spacing";
+import styled from "styled-components";
+import { isAdmin } from "../../../contexts/AnalysisContext";
+import { OutputFileHeader } from "../../../components/OutputFiles/OutputFileHeader";
+
+const JsonOutputWrapper = styled.div`
+  height: 300px;
+  width: 100%;
+  overflow: auto;
+  margin-bottom: ${SPACE_XS};
+  border: solid 1px #bdc3c7;
```

Comment:
Why hard coded colour?

Reply:
Updated in 1f756f8677c8cb5b942e53a44ad93c80f18b6d35